main/common/settings.js

```diff
@@ -90,7 +90,7 @@ const audioInputDeviceId = store.get('audioInputDeviceId');
 (async () => {
   const devices = await audioDevices();
 
```

Comment:
Note https://github.com/wulkano/aperture-node/blob/ef5cd98b9cc6496405bf36177dea79a11cf4952f/index.js#L165, so we could maybe do:

```js
if (!Array.isArray(devices)) {
	// Sentry logging here
	// console.log(devices);

	return
}
```